packages/semantic-ui/src/FieldTemplate/WrapIfAdditionalTemplate.js

```diff
@@ -17,7 +21,8 @@ const WrapIfAdditional = ({
   uiSchema,
   registry,
 }) => {
-  const { RemoveButton } = registry.templates.ButtonTemplates;
+  const uiOptions = getUiOptions(uiSchema);
+  const { RemoveButton } = getTemplate("ButtonTemplates", registry, uiOptions);
```

Comment:
Consider restoring this since we don't need the extra overhead of the call as we can't override them in `uiOptions`. You could add this comment which is in many of the other places (but this one)
```
  // Button templates are not overridden in the uiSchema
```